Steps/BuildPlatform.yml

```diff
@@ -109,14 +117,14 @@ steps:
   displayName: Setup
   inputs:
     script: stuart_setup -c ${{ parameters.build_file }} TOOL_CHAIN_TAG=${{ parameters.tool_chain_tag}} -t ${{ parameters.build_target}} ${{ parameters.build_flags}}
-  condition: and(gt(variables.pkg_count, 0), succeeded())
+  condition: and(and(gt(variables.pkg_count, 0), succeeded()), eq('${{ parameters.skip_setup }}', 'false'))

```

Comment:
@kanechen66, have you verified the condition against true and false scenarios?

Reply:
hi @makubacki 
for `true` case. I tested true case and the setup/update can be skip properly in pipeline
for `false` cae, i tested without setting true from caller.  The setup/update will still be executed in pipeline
![image](https://github.com/user-attachments/assets/6ea11842-7793-4677-93b0-406cee7df828)
Is this ok?
thanks

Reply:
Thanks. The logic in pipelines related to booleans can sometimes be unintuitive so I just wanted to check that both cases were tested as this code is written.